### PR TITLE
I hope it work

### PR DESCRIPTION
**Pull request by Raimund-zim**
To confirm

### CODE REVIEW

File: CSS/style.css [hunk 1]
```diff
@@ -0,0 +1,140 @@
+body {
+	max-width: 1440px;
+	overflow: visible;
+	margin-left: auto;
+	margin-right: auto;
+}
+.header {
+	position: absolute;
```
Comment (shuliakleonid) [inline CSS/style.css]: why absolute?

File: CSS/style.css [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+	margin: 9.5px 149px 9.5px 0px;
+}
+.link__group-1 {
+	display: inline-flex;
+}
+
+.nav__link {
+	display: inline-block;
+	margin: 14px 24.5px 26px 24.5px;
+	font-family: DM Sans;
```
Comment (shuliakleonid) [inline CSS/style.css]: add family font like sens-serif

File: CSS/style.css [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+
+.hero {
+	display: inline-flex;
+	width: 1146px;
+	height: 644px;
+	margin: 190px 100px 0 184px;
+}
+
+.hero__text1 {
+	position: absolute;
```
Comment (shuliakleonid) [inline CSS/style.css]: absolute, you're sure?

File: CSS/style.css [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+	color: #696871;
+}
+.hero__image {
+	margin: 0px 30px 0px 536px;
+}
+.hero__link1 {
+	position: absolute;
+	margin-top: 493px;
+
+	font-family: DM sans, sans-serif;
```
Comment (shuliakleonid) [inline CSS/style.css:117]: your need tiks

File: CSS/style.css [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+	font-family: DM Sans;
+	font-style: normal;
+	font-size: 17px;
+	line-height: 22px;
+	text-align: center;
+	letter-spacing: 0.531474px;
+	color: #ffffff;
+}
+.down--arrow {
+	position: absolute;
```
Comment (shuliakleonid) [inline CSS/style.css]: absolute?

File: index.html [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+			<div class="hero__text1">
+				Grow your subscription business
+			</div>
+		</div>
+		<div class="hero__text2">Outcome-centered products that reduce churn, optimize pricing, and grow your subscription
+			business end-to-end.</div>
+		<div class="hero__links">
+			<form action="https://www.google.com" target="_blank">
+				<button class="hero__link1">Get started</button>
+				<img class="down--arrow" src="/img/Down Arrow.png" alt="Down arrow">
```
Comment (shuliakleonid) [inline index.html]: change class

**Review by shuliakleonid — CHANGES_REQUESTED** [review]

File: CSS/style.css (hunk 3 above)
Comment (Raimund-zim) [reply CSS/style.css]: done

File: CSS/style.css (hunk 2 above)
Comment (Raimund-zim) [reply CSS/style.css]: done

File: CSS/style.css (hunk 1 above)
Comment (Raimund-zim) [reply CSS/style.css]: done

File: CSS/style.css (hunk 3 above)
Comment (Raimund-zim) [reply CSS/style.css]: done

File: CSS/style.css (hunk 4 above)
Comment (Raimund-zim) [reply CSS/style.css:117]: What it means?

File: CSS/style.css (hunk 5 above)
Comment (Raimund-zim) [reply CSS/style.css]: done

File: index.html (hunk 6 above)
Comment (Raimund-zim) [reply index.html]: done

File: CSS/style.css (hunk 4 above)
Comment (shuliakleonid) [reply CSS/style.css:117]: https://developer.mozilla.org/ru/docs/Web/CSS/font-family

**Review by shuliakleonid — APPROVED** [review]

**Comment by shuliakleonid** [comment]
## Good Work!

**Pull request merged**